api-client/src/models/EventModel.ts

```diff
@@ -74,6 +74,17 @@ class EventModel extends ExtendedPageModel {
       body.push(`LOCATION:${location.fullAddress}`)
     }
 
+    if (recurring && date.recurrenceRule) {
+      // rrule.toString adds in the original start date as well which we don't need here
+      const recurrence = date.recurrenceRule
+        .toString()
+        .split('\n')
+        .filter(line => line.startsWith('RRULE:'))[0]
```

Comment:
The reason not to include the start date here is because the imported event then starts at the original start date even if we're long past that. I refactored it to be less finicky.